### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by shubhamkumar648**
alert

### CODE REVIEW

File: component/alert.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link rel="stylesheet" href="../styles/alert.css" />
```
Comment (rahulcs754) [inline component/alert.html]: make one css file and import all css in one file

File: styles/alert.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
+.alert__container {
+
```
Comment (rahulcs754) [inline styles/alert.css]: remove space

File: styles/alert.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
```
Comment (rahulcs754) [inline styles/alert.css]: remove space

File: styles/alert.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    border-radius: 1rem;
+}
+
+.alert-icons{
+
+    margin-right: 1rem;
+
+}
+.alert{
+
```
Comment (rahulcs754) [inline styles/alert.css:9]: same as above

File: styles/alert.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+.alert__container {
+
+    border: 1px solid var(--primary-color);
+    border-radius: 1rem;
+}
+
+.alert-icons{
+
+    margin-right: 1rem;
+
```
Comment (rahulcs754) [inline styles/alert.css]: same as above

File: styles/alert.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+.alert__container {
+
+    border: 1px solid var(--primary-color);
+    border-radius: 1rem;
+}
+
+.alert-icons{
+
```
Comment (rahulcs754) [inline styles/alert.css]: same as above

File: styles/alert.css [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.info {
+    color: var(--info-color);
+
+    background-color: var(--info-bg);
+}
+
+.sucess {
+
```
Comment (rahulcs754) [inline styles/alert.css]: same as above

File: styles/alert.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.warning {
+    color:var(--warning-color);
+    background-color: var(--warning-bg);
+}
+
+.alert-codesnippet-section{
+
+
```
Comment (rahulcs754) [inline styles/alert.css]: same as above

File: styles/alert.css [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    width: 100%;
+    border: 1px solid var(--primary-color);
+    background-color: var(--lightwhite-color);
+    border-radius: 5px;
+    background-color: rgba(255,255,255,1);
```
Comment (rahulcs754) [inline styles/alert.css]: use variable color name

File: styles/alert.css [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+    padding: 0.75rem 1.25rem;
+    border: 1px solid transparent;
+    border-radius: 0.25rem;
+    margin-bottom: 1rem;
+    display: flex;
+}
+
+.info {
+    color: var(--info-color);
+
```
Comment (rahulcs754) [inline styles/alert.css]: same as above

**Review by rahulcs754 — COMMENTED** [review]
I suggest to you some change please check

File: component/alert.html [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+      <main class="main__container p-2">
+        <h3 class="mb-1">Alert</h3>
+        <h6 class="mb-2">Example of Alert</h6>
+        <p class="mb-1">
+          It can be used to notify the user. Variant:
+          danger||success||information|| warning.
+        </p>
+        <div class="alert__container flex-col p-2 mb-2">
+          <div class="alert info">
```
Comment (tanishq20) [inline component/alert.html]: You can give the class name `alert-info` instead of `info`.
This will avoid conflicts in future projects.
(Apply to all)

**Review by tanishq20 — COMMENTED** [review]
Suggested small change.

**Review by vijaytembugade — APPROVED** [review]
Some of the changes are already told to you. Rest things are really good.

File: component/alert.html (hunk 11 above)
Comment (shubhamkumar648) [reply component/alert.html]: thank you done

File: styles/alert.css (hunk 10 above)
Comment (shubhamkumar648) [reply styles/alert.css]: done

File: component/alert.html (hunk 1 above)
Comment (shubhamkumar648) [reply component/alert.html]: done

**Review by rahulcs754 — APPROVED** [review]

**Pull request merged**